tools/PI/DevHome.PI/Controls/ExpandedViewControl.xaml

```diff
@@ -30,22 +30,32 @@
                 </Grid.RowDefinitions>
 
-                <ListView
-                x:Name="NavLinksList" SelectionMode="Single" VerticalAlignment="Stretch" ItemsSource="{x:Bind viewModel.Links}"
-                 SelectedIndex="{x:Bind viewModel.SelectedNavLinkIndex, Mode=TwoWay}" SelectionChanged="{x:Bind viewModel.Navigate}">
-                    <ListView.ItemTemplate>
-                        <DataTemplate>
-                            <StackPanel Orientation="Horizontal">
-                                <TextBlock Text="{Binding IconText}" Style="{x:Null}"
-                                FontFamily="Segoe Fluent Icons" FontSize="12"
-                                HorizontalAlignment="Left" VerticalAlignment="Center"/>
-                                <TextBlock Text="{Binding ContentText}"
-                                Margin="12,0,0,0" HorizontalAlignment="Left" VerticalAlignment="Center"/>
-                            </StackPanel>
-                        </DataTemplate>
-                    </ListView.ItemTemplate>
-                </ListView>
-
+                <StackPanel Orientation="Vertical">
+                    <ListView
+                    x:Name="NavLinksList" SelectionMode="Single" VerticalAlignment="Stretch" ItemsSource="{x:Bind viewModel.Links}"
+                     SelectedIndex="{x:Bind viewModel.SelectedNavLinkIndex, Mode=TwoWay}" SelectionChanged="{x:Bind viewModel.Navigate}">
+                        <ListView.ItemTemplate>
+                            <DataTemplate>
+                                <StackPanel Orientation="Horizontal">
+                                    <TextBlock Text="{Binding IconText}" Style="{x:Null}"
+                                    FontFamily="Segoe Fluent Icons" FontSize="12"
+                                    HorizontalAlignment="Left" VerticalAlignment="Center"/>
+                                    <TextBlock Text="{Binding ContentText}"
+                                    Margin="12,0,0,0" HorizontalAlignment="Left" VerticalAlignment="Center"/>
+                                </StackPanel>
+                            </DataTemplate>
+                        </ListView.ItemTemplate>
+                    </ListView>
+                    <Rectangle x:Name="Separator"
+                        Visibility="{x:Bind viewModel.AppSettingsVisibility, Mode=OneWay}"
+                        RadiusX="{ThemeResource AppBarSeparatorCornerRadius}"
+                        RadiusY="{ThemeResource AppBarSeparatorCornerRadius}"
+                        Height="{ThemeResource AppBarSeparatorWidth}"
+                        Width="200"
+                        Fill="{ThemeResource AppBarSeparatorForegroundThemeBrush}"
+                        Margin="4" />
+                    <Button Width="190" HorizontalAlignment="Center" x:Uid="DetachAppButton" Visibility="{x:Bind viewModel.AppSettingsVisibility, Mode=OneWay}" Command="{x:Bind viewModel.DetachFromProcessCommand}"/>

```

Comment:
Can you use MinWidth instead of Width so text scaling doesn't cut off the text?

Reply:
Ok.